libs/wire-api/src/Wire/API/Routes/Public/Galley.hs

```diff
@@ -731,59 +729,64 @@ type FeatureAPI =
     :<|> FeatureConfigGet 'WithLockStatus 'TeamFeatureSelfDeletingMessages
     :<|> FeatureConfigGet 'WithLockStatus 'TeamFeatureGuestLinks
 
-type FeatureStatusGet featureName =
+type FeatureStatusGet f =
```

Comment:
please, no one-letter names :(

Reply:
Why are you picking on this particular one? The codebase is full of one-letter names. Even for this very purpose (see e.g. `Galley.API.Teams.Features`). I'm actually not sure what the problem is with one-letter names when they stand for very abstract things in a very limited scope (e.g. a single expression, like in this example). Also, the name that was there before isn't even correctly describing what the variable stands for (it's not a "feature name").